twake/backend/node/src/core/platform/services/storage/index.ts

```diff
@@ -0,0 +1,78 @@
+import { createCipheriv, createDecipheriv, randomBytes } from "crypto";
+import { isBuffer } from "lodash";
+import { Stream, Readable } from "stream";
+import { Consumes, logger, TwakeService } from "../../framework";
+import LocalConnectorService, { LocalConfiguration } from "./connectors/local/service";
+import S3ConnectorService, { S3Configuration } from "./connectors/S3/service";
+
+import StorageAPI, { StorageConnectorAPI } from "./provider";
+
+@Consumes([])
+export default class StorageService extends TwakeService<StorageAPI> implements StorageAPI {
+  name = "storage";
+  version = "1";
+
+  globalEncryption: { secret: string | null; iv: string | null } = { secret: null, iv: null };
```

Comment:
should be `private`

Reply:
Wy it is called `global`? This is just `encryptionOptions` composed of a `key` and an `iv`